wire-ios-sync-engine/WireSyncEngine.xcodeproj/xcshareddata/xcschemes/WireSyncEngine.xcscheme

```diff
@@ -128,7 +128,7 @@
          </CommandLineArgument>
          <CommandLineArgument
             argument = "-com.apple.CoreData.ConcurrencyDebug 1"
-            isEnabled = "NO">
+            isEnabled = "YES">
```

Comment:
Should we enable this on default? Does it have impact for production?

Reply:
No impact on prod just when you run the app